content/influxdb/cloud-dedicated/primers/api/v2/_index.md

```diff
@@ -12,14 +13,289 @@ menu:
 influxdb/cloud-dedicated/tags: [write, line protocol]
 ---
 
-Write data with the InfluxDB v2 API `/api/v2/write` endpoint for InfluxDB Cloud:
-  - Send v2 API requests from client libraries, Telegraf, or your own code.
-    - You can't use the InfluxDB `influx` CLI with InfluxDB Cloud Dedicated.
-    - For help finding the best workflow for your situation, [contact the Engineering team](link).
+Use the InfluxDB v2 API for new workloads and existing v2 workloads that you bring to InfluxDB Cloud Dedicated.
+The InfluxDB Cloud Dedicated v2 `/api/v2/write` endpoint works with token authentication
+and existing InfluxDB 2.x tools and code.
+For help finding the best workflow for your situation, [contact Support](mailto:support@influxdata.com).
+
   <!-- v2 SAMPLE CODE -->
-    - Authentication and authorization
-      - Use a Database Token with the `Authorization: Token` scheme or the `Authorization: Bearer`(???) scheme.
-    - Responses
-  - Use client libraries with the v2 API write endpoint
-  - Use Telegraf with the v2 API write endpoint
-  - Use the v2 API write endpoint
\ No newline at end of file
+
+## Authenticate API requests
+
+  <!--TODO: or the `Authorization: Bearer`(???) scheme -->
+  
+InfluxDB requires each write request to be authenticated with a
+[database token](/influxdb/cloud-dedicated/admin/tokens/).
+Use the [`Authorization: Token` scheme](#authenticate-with-the-token-scheme) to provide the token in v2 API requests.
+
+### Authenticate with the Token scheme
+
+To authenticate using the `Token` scheme, pass the `Authorization` header with the `Token` scheme and a
+[database token](/influxdb/cloud-dedicated/admin/tokens/) that has sufficient read/write permissions to the database.
+
+#### Syntax
+
+```http
+Authorization: Token DATABASE_TOKEN
+```
+
+#### Example
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[curl](#curl)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+
+The following example shows how to use the **cURL** command line tool
+and the InfluxDB Cloud Dedicated v2 API to write line protocol data to a database:
+
+```sh
+{{% get-shared-text "api/cloud-dedicated/token-auth-v2-write.sh" %}}
+```
+
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
```

Comment:
- Is there a reason to use code-tabs here when there's only one tab?
- Does this render correctly with the non-code content inside of the `code-tab-content` block?